mne/decoding/transformer.py

```diff
@@ -33,20 +33,37 @@ class Scaler(TransformerMixin):
     ----------
     info : instance of Info
         The measurement info
+    scaler : string, defaults to "standard"
```

Comment:
this variable should also accept being a dict. See `scalings` in other functions